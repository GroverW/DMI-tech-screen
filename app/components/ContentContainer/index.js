/**
 *
 * ContentContainer
 *
 */

import styled from 'styled-components';

const ContentContainer = styled.div`
  padding: 10px 15px;
  margin: 0px auto;
  width: 600px;
  box-shadow: 0px 4px 2px rgba(0, 0, 0, 0.2);
  border-radius: 5px;
`;

export default ContentContainer;
