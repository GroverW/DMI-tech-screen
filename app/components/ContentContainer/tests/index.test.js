/**
 *
 * Tests for ContentContainer
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from 'react-testing-library';
// import 'jest-dom/extend-expect'; // add some helpful assertions

import ContentContainer from '../index';

describe('<ContentContainer />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<ContentContainer />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<ContentContainer />);
    expect(firstChild).toMatchSnapshot();
  });
});
