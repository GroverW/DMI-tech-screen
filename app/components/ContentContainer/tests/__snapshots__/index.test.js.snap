// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentContainer /> Should render and match the snapshot 1`] = `
<div
  class="ContentContainer-sc-18efx5u-0 lpiSfa"
/>
`;
